src/Node/Render.php

```diff
@@ -88,11 +88,13 @@ protected function addGetTemplate(Twig_Compiler $compiler) {
     else {
       $compiler->raw('$passed_variables = ')->subcompile($this->getNode('variables'))->raw(";\n");
       $compiler->raw('$variables = array_merge($defaults, $passed_variables)')->raw(";\n");
-      $compiler->raw('unset($variables[\'attributes\'], $variables[\'title_attributes\'], $variables[\'content_attributes\'])')->raw(";\n");
-
       $compiler->raw(<<<'EOD'
 
-        foreach (['attributes', 'title_attributes', 'content_attributes'] as $name) {
+        foreach ($variables as $name => $value) {
+          if (FALSE === strpos($name, 'attributes')) {
```

Comment:
Just for curiosity: is there a purpose to use yoda notation here or just to stick some kind of standards? 
But is it not against Drupal standards?

I'm asking because generally we're not doing it, no? 🤔

Reply:
You're right, we usually compare the actual value to a expected value from left to right.

This is a habit grown out of the following more common pattern:

```diff
-          if (FALSE === strpos($name, 'attributes')) {
+          if (FALSE === $pos = strpos($name, 'attributes')) {
```
(Assigning the position to `$pos` and using it within the condition if it's true, but comparing the value at the same time.  The assignment would require parentheses otherwise.)